src/tests/ftest/osa/osa_offline_reintegration.py

```diff
@@ -9,6 +9,7 @@
 from osa_utils import OSAUtils
 from test_utils_pool import TestPool
 from write_host_file import write_host_file
+import queue as queue
```

Comment:
(pylint-useless-import-alias) Import alias does not rename original package
(pylint-unused-import) Unused queue imported as queue